src/tpl/components/main-header/index.nunj

```diff
@@ -0,0 +1,6 @@
+<header class="main-header fixed-top">
+	<div class="container d-flex">
```

Comment:
ccheck nesting

Reply:
Nesting seems to be OK. Two spaces are there